backends/conrod_vulkano/src/lib.rs

```diff
@@ -485,15 +467,25 @@ impl Renderer {
                             continue;
                         }
                         if let Some(image) = image_map.get(&image_id) {
-                            let desc_image = Arc::new(
-                                tex_descs
-                                    .next()
-                                    .add_sampled_image(
-                                        ImageView::new(image.image_access.clone()).unwrap(),
-                                        self.sampler.clone(),
-                                    )?
-                                    .build()?,
-                            );
+                            // ??????????? is there any reason to use self.tex_descs
```

Comment:
It's been so long enough that I don't recall what's going on here - there's a chance this code is old enough that `FixedSizeDescriptorSetsPool` was the only option at that time, and it's slowly been adapted ever since.